tests/suites/test_suite_ssl.function

```diff
@@ -6,6 +6,39 @@
 #include <mbedtls/certs.h>
 #include <mbedtls/timing.h>
 
+typedef struct handshake_test_options
+{
+    const char *cipher;
+    int version;
+    int pk_alg;
+    data_t *psk_str;
+    int dtls;
+    int serialize;
+    int mfl;
+    int cli_msg_len;
+    int srv_msg_len;
+    int expected_cli_fragments;
+    int expected_srv_fragments;
+    int renegotiate;
+    int legacy_renegotiation;
+} handshake_test_options;
```

Comment:
We should start adding some simple descriptions of these parameters in the future